src/components/DoiMetadata/DoiMetadata.tsx

```diff
@@ -80,7 +80,7 @@ const DoiMetadata: React.FunctionComponent<Props> = ({ item }) => {
 
   const title = () => {
     const router = useRouter()
-    if (router == null || router.pathname === '/' || router.pathname === '/person/[person]') {
+    if (router == null || router.pathname === '/' || router.pathname === '/people/[person]') {
```

Comment:
Appreciate this isn't new as such, but a general thought, I wonder if we shouldn't pass in as a prop to determine title.
For Organisations I was looking at just saying explicitly whether it was a "DetailsPage" via a boolean, that way you just tell it when it's this, rather than relying on the router